pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -638,6 +638,7 @@ self: super:
       if (!isDarwin)
       then {
         outputs = [ "out" "dev" ];
+        patches = [ ./dont-create-logdir.patch ];
```

Comment:
The commit message has a rather lengthy explanation already, I didn't want to bloat the code.

`dont-create-logdir-during-build` would be a better name for the patch though.

Reply:
Yes, the commit message is very helpful. But its even nicer to have the "why" at a glance, without having to dig into it. It doesn't have to go into any detail, just `# the default logdir is outside the nix store and will not be accessible`.

Reply:
@Atemu This should be explained in the code, not the commit message:

1. Understanding the new state of the code in isolation should be in code comments
2. Understanding the new state in relation to the old state should be in the commit message.

This cases falls into case 1, because someone can wonder what the patch is for at any point. Only answers to questions like "I understand the new and old way of doing things, but why is the new way better?" go in commit messages.